### PR TITLE
Add delete functions for reviews. Also add svg delete icon and use it…

### LINKED ISSUE #1097: Ability for an author of draft and submitted reviews to delete either

### PR DESCRIPTION
**Pull request by frjo**
… on reviews and on submission.

Fixes #1097

### CODE REVIEW

**Comment by frjo** [comment]
@Parbhat  Copied most things from your pr #1145

File: opentech/apply/review/templates/review/review_confirm_delete.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% extends "base-apply.html" %}
+
+{% block title %}Deleting: {{object.title }}{% endblock %}
```
Comment (Parbhat) [inline opentech/apply/review/templates/review/review_confirm_delete.html]:
```suggestion
{% block title %}Deleting: {{object}}{% endblock %}
```

File: opentech/apply/review/templates/review/review_confirm_delete.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+{% block content %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h2 class="heading heading--no-margin">Deleting: {{ object.title }}</h2>
```
Comment (Parbhat) [inline opentech/apply/review/templates/review/review_confirm_delete.html]:
```suggestion
        <h2 class="heading heading--no-margin">Deleting: {{ object }}</h2>
```

**Comment by frjo** [comment]
Yes, author. Manage to forget that. Tips how to implement that?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Parbhat** [comment]
We can use [UserPassesTestMixin](https://docs.djangoproject.com/en/2.2/topics/auth/default/#django.contrib.auth.mixins.UserPassesTestMixin) in ReviewDeleteView to check requesting user is same as review's author.

**Comment by frjo** [comment]
@Parbhat Thank for the UserPassesTestMixin suggestion. I believe I have a working implementation now that allows review author to delete.

**Comment by fourthletter** [comment]
Delete functions for reviews was available and functional during test. Thanks!

**Pull request merged**